### DISCUSSION TITLE
Disable keyboard navigation

### DISCUSSION THREAD

**Discussion by n1k1c4** [gpbl/react-day-picker#1462] [Ideas]
We need to add the ability to disable keyboard navigation. It seems like a trivial thing. Keyboard navigation is not always appropriate.

**Comment by gpbl** [comment]
Hi @n1k1c4 – some more context? Why do you want to disable it?

**Comment by mantir** [comment]
i got two inputs for a date range and want to go with tab from the start input to the end input. but when the daypicker is open, tab cycles through all the buttons in the calendar.
